src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -576,6 +576,24 @@ public EmailWrapper generateInstructorCourseJoinEmail(AccountAttributes inviter,
         return email;
     }
 
+    /**
+     * Generates the course register email for the given {@code user} in {@code course}.
+     */
+    public EmailWrapper generateUserCourseRegisterEmail(AccountAttributes user, CourseAttributes course) {
+        String emailBody = Templates.populateTemplate(EmailTemplates.USER_COURSE_REGISTER,
+                "${userName}", SanitizationHelper.sanitizeForHtml(user.getName()),
+                "${courseId}", SanitizationHelper.sanitizeForHtml(course.getId()),
+                "${courseName}", SanitizationHelper.sanitizeForHtml(course.getName()),
+                "${googleId}", SanitizationHelper.sanitizeForHtml(user.getGoogleId()),
+                "${supportEmail}", Config.SUPPORT_EMAIL);
+
+        EmailWrapper email = getEmptyEmailAddressedToEmail(user.getEmail());
+        email.setSubject(String.format(EmailType.USER_COURSE_REGISTER.getSubject(),
+                course.getId(), course.getName()));
```

Comment:
The contents of the email subject do not match (currently it shows the course name as "Course ID:")